src/io/flutter/toolwindow/ToolWindowBadgeUpdater.java

```diff
@@ -10,47 +10,81 @@
 import com.intellij.openapi.wm.ToolWindow;
 import com.intellij.openapi.wm.ToolWindowId;
 import com.intellij.openapi.wm.ToolWindowManager;
-import com.intellij.ui.BadgeIcon;
 import io.flutter.run.common.RunMode;
 import io.flutter.run.daemon.FlutterApp;
 
-import java.awt.Color;
-import java.awt.Paint;
+import javax.swing.*;
```

Comment:
I was under the impression that we avoid * imports, but I'm not sure if that's enforced somewhere in java settings.

Reply:
Interesting! This is what the IDE did on-save. Doing a quick grep I see we've got 104+ `*` imports in 74+ files so we're definitely not consistently avoiding them. 

That could be this inspection?

https://www.jetbrains.com/help/inspectopedia/OnDemandImport.html

Happy to discuss enabling!

Reply:
Thinking a bit about it, I tend to agree. Wildcard imports are probably best to avoid. Could you add some discussion to https://github.com/flutter/flutter-intellij/issues/8098? 

We do have a checked in inspection set and think we should probably enable it there but we should discuss. It would also be great to get the inspections run in the CI, if not as part of the verifier, as a separate task. I've poked at that a bit but got distracted. (Some info here: https://www.jetbrains.com/help/idea/command-line-code-inspector.html#inspection-profiles.)